### PR TITLE
docstrings

### PR DESCRIPTION
**Pull request by tidalf**

### CODE REVIEW

File: irbt/cloud.py [hunk 1]
```diff
@@ -12,8 +17,19 @@
...
 class Cloud:
+    """
+    Cloud class.
+
+    It allow interactions with the irbt cloud api
+    """
 
     def __init__(self, username=None, password=None):
+        """
+        Initiliaze usefull params.
```
Comment (showi) [inline irbt/cloud.py]: typo

File: irbt/robot.py [hunk 2]
```diff
@@ -8,23 +13,45 @@
...
         stop: None
         pause: None
         dock: None
-        dockz: None
 
         def __init__(self, robot):
+            """
+            Declare the partial func.
+
+            Used to gate what command are sent through mqtt
```
Comment (showi) [inline irbt/robot.py]: open the gate

File: irbt/robot.py [hunk 3]
```diff
@@ -8,23 +13,45 @@
...
             self.stop = functools.partial(robot._cmd, "stop")
             self.pause = functools.partial(robot._cmd, "pause")
             self.dock = functools.partial(robot._cmd, "dock")
             self.status = functools.partial(robot._cmd, "status")
 
     def __init__(self, cloud=None, id=None, output_raw=None):
+        """
+        Initialize the robot instance.
+
+        We first check if we have a cloud the we gather
```
Comment (showi) [inline irbt/robot.py]: the we is better than monday

File: irbt/robot.py (hunk 2 above)
Comment (tidalf) [reply irbt/robot.py]: gate is cool there no ?

File: irbt/robot.py [hunk 4]
```diff
@@ -76,6 +118,11 @@ def missions(self):
         ).content.decode("utf-8")
 
     def evac_history(self):
+        """
+        Return logs of evacuations.
```
Comment (showi) [inline irbt/robot.py]: for the end of the world as we know it

File: irbt/cloud.py (hunk 1 above)
Comment (tidalf) [reply irbt/cloud.py]: done

File: irbt/robot.py (hunk 2 above)
Comment (showi) [reply irbt/robot.py]: ok dunno :)

File: irbt/robot.py (hunk 3 above)
Comment (tidalf) [reply irbt/robot.py]: done

**Pull request merged**